rules-tests/Php71/Rector/FuncCall/CountOnNullRector/Fixture/false_true_class.php.inc

```diff
@@ -27,13 +27,13 @@ use stdClass;
 function countables2()
 {
     $values = false;
-    $count = is_array($values) || $values instanceof \Countable ? count($values) : 0;
+    $count = is_array($values) || $values instanceof \Countable ? count($values) : (is_null($values) ? 0 : 1);
```

Comment:
false is a constant value, I think replace it directly to 1 should be the way to go, so should be:

```suggestion
    $count = 1;
```

other fixture should applies to for constant value as well.